care/emr/tests/test_diagnosis_api.py

```
@@ -0,0 +1,621 @@
+            facility=self.facility,
+            organization=self.organization,
+        )
+        diagnosis = self.create_diagnosis(encounter=encounter, patient=self.patient)
+
+        url = self._get_diagnosis_url(diagnosis.external_id)
+        diagnosis_data_updated = model_to_dict(diagnosis)
+        diagnosis_data_updated["severity"] = "mild"
+        diagnosis_data_updated["code"] = self.valid_code
+
+        response = self.client.put(url, diagnosis_data_updated, format="json")
+        print(response.json())
```

Ruff (0.8.2)

`print` found

(T201)